ci-operator/step-registry/baremetal/lab/pre/firewall/baremetal-lab-pre-firewall-commands.sh

```diff
@@ -13,6 +13,29 @@ SSHOPTS=(-o 'ConnectTimeout=5'
   -o LogLevel=ERROR
   -i "${CLUSTER_PROFILE_DIR}/ssh-key")
 
+setup_proxy() {
+  proxy="$(<"${CLUSTER_PROFILE_DIR}/proxy")"
+  cat <<EOF > "${SHARED_DIR}/proxy-conf.sh"
+  export HTTP_PROXY=${proxy}
+  export HTTPS_PROXY=${proxy}
+  export NO_PROXY="localhost,127.0.0.1"
+
+  export http_proxy=${proxy}
+  export https_proxy=${proxy}
+  export no_proxy="localhost,127.0.0.1"
+EOF
+}
+# shellcheck disable=SC2154
+if [ "${masters}" -eq 1 ]; then
+  setup_proxy
+fi
+
+if [ "${ipv6_enabled:-}" == "true" ]; then
```

Comment:
This is not fine in dual stack cases. When doing a single stack install, we must have CLUSTER_WIDE_PROXY= true

Reply:
Changes have been implemented.